src/utils/mixins.py

```diff
@@ -81,11 +80,18 @@ class AnyTeamFacilitatorRequiredMixin(BaseTeamPermRequiredMixin):
 class AnyTeamPosRequiredMixin(BaseTeamPermRequiredMixin):
     """Mixin for views available to anyone with a "camps.<team>_team_pos" permission for any team.
 
-    Currently used to control access to the backoffice POS list, POS Transaction List, POS sale list, POS product list, and POS Product Cost List views."""
+    Currently used to control access to the backoffice POS list, POS Transaction List, POS sale list, POS product list, and POS Product Cost List views.
+    """
 
     perm = "pos"
 
 
+class TeamTaskerRequiredMixin(BaseTeamPermRequiredMixin):
```

Comment:
this needs to check for the tasker perm for the specific team in question, like we do in https://github.com/bornhack/bornhack-website/blob/master/src/teams/views/mixins.py#L58-L62